docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -268,6 +271,21 @@
       shell: KUBECONFIG=/etc/kubernetes/admin.conf /usr/sbin/helm upgrade --install deployment-manager {% if helm_chart_overrides is defined %}--values {{ helm_chart_overrides }}{% endif %} {{ manager_chart }}
       when: helmv2_dm_exists.rc != 0
 
+    - name: Check if DM-monitor playbook file exists in path specified
+      stat:
+        path: "{{ dm_monitor_playbook }}"
+      register: dm_monitor_playbook_file
+
+    - name: Fail if dm-monitor playbook does not exist in path specified
```

Comment:
+1

Reply:
We can remove the "Check if DM-monitor playbook file exists in path specified" and "Fail if dm-monitor playbook does not exist in path specified". 

Then, when installing DM-Monitor you should add:

# Install or Upgrade DM-Monitor
    - name: "Install or Upgrade DM-monitor"
      shell: "ansible-playbook {{ dm_monitor_playbook }}"
      become: yes
      **ignore_errors: yes
      when: dm_monitor_playbook**

I'd also rename this task to "Install or Upgrade.." since it can be used also for a fresh installl.

Reply:
The **ignore_errors** will also prevent the DM playbook from failing if an error occurs during the execution of the DM-Monitor playbook.

Reply:
@enzocandotti dm_monitor_playbook is always defined since we are setting default value.
Do we still need dm_monitor_playbook?